client/galaxy/scripts/components/RuleCollectionBuilder.vue

```diff
@@ -338,7 +338,7 @@
                 <button @click="resetRulesAndState" :title="titleReset" class="creator-reset-btn btn rule-btn-reset">
                     {{ l("Reset") }}
                 </button>
-                <button @click="createCollection" :title="titleFinish" class="create-collection btn btn-primary rule-btn-okay" v-bind:class="{ disabled: !validInput }">
+                <button @click="createCollection" :title="titleFinish" class="create-collection btn btn-secondary btn-primary" v-bind:class="{ disabled: !validInput }">
```

Comment:
having both btn-primary and btn-secondary classes on a button is superfluous I think

Reply:
disregard, was amended in subsequent commit